src/SmartComponents/BaselinesTable/BaselinesTable.js

```diff
@@ -91,22 +107,31 @@ class BaselinesTable extends Component {
         return table;
     }
 
+    createModalRows() {
```

Comment:
What makes a modal row a modal row vs. in the normal table?

Reply:
At the moment, modalRows have the checkboxes and don't have the kebab. The normal table has kebab but no checkboxes.

Reply:
modalRows have checkboxes and no kebabs